### DISCUSSION TITLE
remix vs nextjs

### DISCUSSION THREAD

**Discussion by ziat1988** [remix-run/remix#574] [Q&A] [answered]
Hi,
Just a dump question because it's not clear for me to understand the purpose of remix (from the docs)
It seems that remix do the ssr but what is the different between remix and nextjs if I use reactjs ?
Thanks

**Comment by arvigeus** [comment]
https://medium.com/wayf/next-js-and-remixa-brief-comparison-8b542cb2e4f4

**Comment by sergiodxa** [comment]
I like this article https://blog.plasmic.app/posts/remix-for-nextjs-developers/

**Comment by machour** [comment, marked as answer]
There is now an official blog entry for this comparison: https://remix.run/blog/remix-vs-next
